deepchem/data/data_loader.py

```diff
@@ -1740,4 +1742,4 @@ def _featurize_shard(self, shard):
                 "Unknown key in yaml file. Please check format for correctness."
             )
         x = DFTEntry.create(e_type, true_val, systems)
```

Comment:
Why is only 1 entry being created for an entire shard? This step should return as many DFTEntry objects as there were datapoints in the original shard